File: Fact_Difference_Summation/Diff.c
/*
    Auhtor : Pakshal Jain
    Date : 06/10/2022
    Program : TO Find Difference Between Summation of Factors and Nonfactors of Entered Number
*/

//Required Libraries
#include<stdio.h>

//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

//Function To Find Difference
int FactDiff(int No)
{
    int i = 0;
    int sum = 0;
    int sum2 = 0;
    int diff = 0;

    for(i = 1;i < No;i++)
    {
        if(No % i == 0)
        {
            sum = sum + i;
        }
        else if(No % i != 0)
        {
            sum2 = sum2 + i;
        }
    }

    diff = sum - sum2;

    return diff;
}

/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////

int main() 
{
    int No = 0;
    int iret = 0;

    printf("Jay Ganesh...\n");
    
    printf("Enter one Number\n");
    scanf("%d",&No);

    //Call TO Difference Function
    iret = FactDiff(No);

    printf("Difference Between Summation of Factors and Nonfactors is : %d",iret);

    return 0;
}

File: Fact_Difference_Summation/Diff.java
/*
    Author : Pakshal Jain
    Date : 06/10/2022
    Program : To Calculate Differece Between Summation of Factors and Nonfactors of Entered Number
*/

//Required Libraries
import java.lang.*;
import java.util.*;

//Class Declaration
class Find
{
    //Characteristics of Class
    private int No = 0;

    //Constructor of Class
    Find(int no)
    {
        this.No = no;
    }
    //Behaviour of Class
    int factdiff()
    {
        int i = 0;
        int sum = 0;
        int sum2 = 0;
        int diff = 0;

        for(i = 1;i < this.No;i++)
        {
            if(this.No % i == 0)
            {
                sum = sum + i;
            }
            else if(this.No % i != 0)
            {
                sum2 = sum2 + i;
            }
        }

        diff = sum - sum2;

        return diff;
    }
}//End of Class

//Main Class
class Diff
{
    public static void main(String args[])
    {
        int No = 0;
        int iret = 0;

        System.out.println("Jay Ganesh...");
        
        Scanner sobj = new Scanner(System.in);
        
        System.out.println("Enter One Number");
        No = sobj.nextInt();

        //Creating object of Find Class
        Find fobj = new Find(No);
        //Calling Behvaiour of Find Class
        iret = fobj.factdiff();

        System.out.println("Difference Between Summation of Nonfactors and factors of Entered Number is : "+iret);
    }
}